bsp/stm32/stm32f103-atk-warshipV3/applications/main.c

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2006-2018, RT-Thread Development Team
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Change Logs:
+ * Date           Author       Notes
+ * 2019-03-08     obito0   first version
+ */
+
+#include <rtthread.h>
+#include <rtdevice.h>
+#include <board.h>
+
+/* defined the LED0 pin: PB5 */
+#define LED0_PIN    GET_PIN(B, 5)
+/* defined the LED0 pin: PE5 */
+#define LED1_PIN    GET_PIN(E, 5)
+
+int main(void)
+{
+    int count = 1;
+    /* set LED0 pin mode to output */
+    rt_pin_mode(LED0_PIN, PIN_MODE_OUTPUT);
+	/* set LED1 pin mode to output */
+    rt_pin_mode(LED1_PIN, PIN_MODE_OUTPUT);
+
+    while (count++)
+    {
+        rt_pin_write(LED0_PIN, PIN_HIGH);
+		rt_pin_write(LED1_PIN, PIN_HIGH);
```

Comment:
Please take care of indention.

Reply:
The indentation in RT-Thread is space character, not TAB character. Please correct all of them in this PR files.